src/Raven.Server/Web/System/StudioHandler.cs

```diff
@@ -120,5 +123,26 @@ public Task RavenRoot()
             HttpContext.Response.StatusCode = 301;
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/c-sharp-index-definition", "GET")]
```

Comment:
I know, but know there is chaos, and this is clearly studio related endpoint.

Another option is to move this action under
/databases/*/indexes/c-sharp-index-definition
like here
https://github.com/ayende/ravendb/blob/v4.0/src/Raven.Server/Documents/Handlers/IndexHandler.cs#L54